identity_iota_core/src/rebased/iota/well_known_networks.rs

```diff
@@ -0,0 +1,73 @@
+use iota_sdk::types::base_types::ObjectID;
+use phf::{phf_map, Map};
+
+/// A Mapping `NetworkID` -> metadata needed by the library.
```

Comment:
Heavily nitpicking.

Maybe we should use something like "`network_id`" or just "network id" here. Was searching for the type/enum "`NetworkID`" in the code after seeing this. :D

```suggestion
/// A Mapping `network_id` -> metadata needed by the library.
```